int64 req_frame_id
---
int64 pulse_difference
int64 balance_pulses
